pkg/system/templates/etc/systemd/system/kubelet.service.d/30-kube-reserved.conf

```diff
@@ -1,2 +1,4 @@
 [Service]
+{{- if and .Node.ReservedCPU .Node.ReservedMemory }}
```

Comment:
So, if one of them is unavailable we're assuming the instance type is unknown and we won't configure any reservation. Cool! :smile: :+1:

Reply:
Yep right ...